src/testFixtures/java/org/opensearch/knn/KNNRestTestCase.java

```diff
@@ -770,6 +765,37 @@ public List<List<String>> bulkSearch(String testIndex, String testField, float[]
         return searchResults;
     }
 
+    // Method that waits till the health of nodes in the cluster goes green
+    public void waitForClusterHealthGreen(String numOfNodes) throws IOException {
+        Request waitForGreen = new Request("GET", "/_cluster/health");
+        waitForGreen.addParameter("wait_for_nodes", numOfNodes);
+        waitForGreen.addParameter("wait_for_status", "green");
+        client().performRequest(waitForGreen);
+    }
+
+    // [BWCTests] Add KNN docs into a KNN index by providing the initial documentID and number of documents
```

Comment:
Remove the [BWCTests] - if it is specific to BWCTests it shouldnt be in this class.

Reply:
It is a generic method. I will update the comments and method name